app/src/display/widgets/output_status.c

```diff
@@ -39,24 +39,24 @@ static struct output_status_state get_state(const zmk_event_t *_eh) {
 }
 
 static void set_status_symbol(lv_obj_t *label, struct output_status_state state) {
-    char text[9] = {};
+    char text[10] = {};
 
     switch (state.selected_endpoint) {
     case ZMK_ENDPOINT_USB:
-        strcat(text, LV_SYMBOL_USB "   ");
+        strcat(text, LV_SYMBOL_USB);
         break;
     case ZMK_ENDPOINT_BLE:
         if (state.active_profile_bonded) {
             if (state.active_profile_connected) {
-                snprintf(text, sizeof(text), LV_SYMBOL_WIFI "%i " LV_SYMBOL_OK,
-                         state.active_profile_index);
+                snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_OK,
+                         state.active_profile_index + 1);
```

Comment:
I'm all for this... but definitely risks confusion because of the indexing discrepancy between keymaps and the display... We do already not the indexes are zero based in the bluetooth docs... I suppose we just do it?

Once we have studio, we can hide it behind a UI anyways...

Reply:
That is a fair point, but I think 0-indexing is relatively uncommon and strange in a user-facing application like this. As you said, we note it, so hopefully, we shouldn't have too many issues. Studio will also abstract all that away too!